ui/apps/platform/src/Containers/NetworkGraph/NetworkGraphPage.css

```diff
@@ -34,12 +34,12 @@
 
 /* Create a stacking context that is higher than the context for the react-topology component
    that is rendered below the toolbar, so that the search filter dropdown will render above the graph when open */
-[data-testid="network-graph-toolbar"] {
+[data-testid='network-graph-toolbar'] {
```

Comment:
🤔

Reply:
Visual Studio Code sometimes does the opposite in TypeScript files which prettier does fight.

If it is the editor or AI then please do fight it. I am not aware of any change in our config and it would mess me up with Find in Files.

Reply:
In fact, ESLint now supports CSS files (although it delegates style like this to prettier).

Reply:
@pedrottimark Would it make sense for us to include scripts for something like `lint:prettier` and `lint:all` where it runs `npm run lint && npm run lint:prettier`?